aws/resource_aws_emr_cluster.go

```diff
@@ -485,6 +504,181 @@ func resourceAwsEMRCluster() *schema.Resource {
 	}
 }
 
+func InstanceFleetConfigSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
```

Comment:
Since this new Instance Fleet support does not implement the [`ModifyInstanceFleet` API call](https://docs.aws.amazon.com/emr/latest/APIReference/API_ModifyInstanceFleet.html), all `Optional: true`/`Required: true` parent and children attributes in this schema must be marked `ForceNew: true`.